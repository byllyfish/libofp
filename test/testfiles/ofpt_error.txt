---
type: OFPT_ERROR
version: 1
msg:
  type: 1
  code: 1
  data: '0123456789'
---
type: OFPT_ERROR
msg:
  type: 1
  code: 1
  data: '0123456789'
---
type: OFPT_ERROR
version: 2
msg:
  type: 1
  code: 1
  data: '0123456789'
---
type: OFPT_ERROR
version: 3
msg:
  type: 1
  code: 1
  data: '0123456789'
---
type: OFPT_ERROR
version: 4
msg:
  type: 1
  code: 1
  data: '0123456789'
---
