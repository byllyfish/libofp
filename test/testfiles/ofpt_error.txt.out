---
event:           LIBOFP_LOOPBACK
msg:             
  data:            0101001100000000000100010123456789
...
---
event:           LIBOFP_LOOPBACK
msg:             
  data:            0001001100000000000100010123456789
...
---
event:           LIBOFP_LOOPBACK
msg:             
  data:            0201001100000000000100010123456789
...
---
event:           LIBOFP_LOOPBACK
msg:             
  data:            0301001100000000000100010123456789
...
---
event:           LIBOFP_LOOPBACK
msg:             
  data:            0401001100000000000100010123456789
...
